spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/hive/TestHivePropertiesConverter.java

```diff
@@ -96,6 +96,24 @@ void testExternalTable() {
         ImmutableMap.of(HivePropertiesConstants.SPARK_HIVE_EXTERNAL, "true"), hiveProperties);
   }
 
+  @Test
+  void testLocation() {
```

Comment:
could you add the logic to transform the Spark location and Gravitino location?  the logic seems duplicate at the first time, but I think it's necessary because we couldn't assume  `HivePropertiesConstants.SPARK_HIVE_LOCATION` and `HivePropertiesConstants.GRAVITINO_HIVE_LOCATION` are same string.